Impl/sdl_rpc_plugin

BSolonenko opened the pull request:
This PR is **ready** for review.

### Risk
This PR makes **no** API changes.

### Summary
This PR provides changes to HMI and mobile commands.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

Commit:
Copywriting added.

In 'src/components/application_manager/rpc_plugins/sdl_rpc_plugin/'
folder added copywriting for next files:
include/sdl_rpc_plugin/sdl_rpc_plugin.h
src/commands/hmi/navi_stop_stream_request.cc
src/commands/hmi/on_tts_reset_timeout_notification.cc
src/commands/mobile/get_system_capability_request.cc
src/commands/mobile/get_system_capability_response.cc
src/commands/mobile/get_way_points_request.cc
src/commands/mobile/get_way_points_response.cc
src/commands/mobile/subscribe_way_points_request.cc
src/commands/mobile/subscribe_way_points_response.cc
src/commands/mobile/unsubscribe_way_points_request.cc
src/commands/mobile/unsubscribe_way_points_response.cc
src/sdl_rpc_plugin.cc

Commit:
Refactoring CommandImpl, CommandRequestImpl and CommandResponseImpl

In CommandImpl added rpc_service_, hmi_capabilities_ and policy_handler_
fields. Also changed constructors CommandImpl, CommandRequestImpl and
CommandResponseImpl.

Commit:
Refactoring all mobile commands.

In all mobile command added rpc_service_, hmi_capabilities_ and
policy_handler_ fields. Also changed all constructors.

Commit:
Refactoring all hmi commands.

In all hmi command added rpc_service_, hmi_capabilities_ and
policy_handler_ fields. Also changed all constructors.

BSolonenko commented:
@LuxoftAKutsan @LitvinenkoIra Please review.

LuxoftAKutsan commented:
Greate!
Now please replas]ce all usages of passed parameters in implementation from application manager proxy to indirect call

Commit:
Replaced all usages of passed parameters in implementation.

Replaced all usages of passed parameters in implementation from
application manager proxy to indirect call

BSolonenko commented:
@LuxoftAKutsan Replaced in https://github.com/LuxoftAKutsan/sdl_core/pull/270/commits/4cf47e98518cd14cb3efd7bc61c6b19d020f0cac
